rbi/annotations/lhm.rbi

```diff
@@ -1,6 +1,9 @@
 # typed: true
 
 module Lhm
+  # TODO: may not be the right fix
+  class Connection; end
```

Comment:
I'm not sure why it's new, but `bundle exec repo check static lhm` was failing with:

```
Error: rbi/annotations/lhm.rbi:7: Unable to resolve constant Connection https://srb.help/5002
     7 |  sig { returns(Lhm::Connection) }
                        ^^^^^^^^^^^^^^^

rbi/annotations/lhm.rbi:22: Unable to resolve constant Connection https://srb.help/5002
    22 |  sig { returns(Lhm::Connection) }
                        ^^^^^^^^^^^^^^^

rbi/annotations/lhm.rbi:31: Unable to resolve constant Connection https://srb.help/5002
    31 |  sig { params(adapter: Lhm::Connection).returns(Lhm::Connection) }
                                ^^^^^^^^^^^^^^^

rbi/annotations/lhm.rbi:31: Unable to resolve constant Connection https://srb.help/5002
    31 |  sig { params(adapter: Lhm::Connection).returns(Lhm::Connection) }
                                                         ^^^^^^^^^^^^^^^

rbi/annotations/lhm.rbi:60: Unable to resolve constant Connection https://srb.help/5002
    60 |  sig { params(table_name: T.any(Symbol, String), connection: Lhm::Connection).returns(Lhm::Table) }
                                                                      ^^^^^^^^^^^^^^^

rbi/annotations/lhm.rbi:102: Unable to resolve constant Connection https://srb.help/5002
     102 |  sig { params(table: Lhm::Table, connection: T.nilable(Lhm::Connection)).void }
                                                                  ^^^^^^^^^^^^^^^

rbi/annotations/lhm.rbi:130: Unable to resolve constant Connection https://srb.help/5002
     130 |  sig { returns(T.nilable(Lhm::Connection)) }
                                    ^^^^^^^^^^^^^^^
```

Reply:
This shouldn't be needed, I can see that `Lhm::Connection` is generated locally. Can you add a breakpoint and take a look at the RBI generated by tapioca during a `check static lhm`?